test/suites/standard1/configuration.robot

```diff
@@ -70,3 +108,22 @@ Setup With Debug Log Level
     ${merged}=    Extend MicroShift Config    ${BAD_LOG_LEVEL}
     Upload MicroShift Config    ${merged}
     Restart MicroShift
+
+Setup With Unknown Audit Log Profile
+    [Documentation]    Set audit log profile to unknown value
+    ${merged}=    Extend MicroShift Config    ${BAD_AUDIT_PROFILE}
+    Upload MicroShift Config    ${merged}
+    Stop MicroShift
+    Start MicroShift
```

Comment:
Any reason here it's Stop + Start but in other places it's Restart?

Reply:
This was a pain to realize, but the Restart MicroShift keyword fails the test if MicroShift doesn't restart, which is the expected outcome here.